### ISSUE TITLE
quic-go: does own logging

### ISSUE THREAD

**Issue by AskAlexSharov** [erigontech/erigon#12758]
```
2024/11/18 12:58:55 failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 7168 kiB, got: 416 kiB). See https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes for details.
```

**Comment by lystopad** [comment]
Reviewed against `release/3.5` and `release/3.6` (both pin `quic-go v0.59.1`). Closing as no longer relevant for default operation.

**Where the line comes from:** quic-go emits it via Go's stdlib `log` (raw stderr, not erigon's logger) from `wrapConn`, and only when the OS won't grant its UDP socket the desired ~7 MiB receive buffer.

**Why it no longer shows up:**
- QUIC is unreachable in a stock node. Caplin's libp2p is TCP-only (`cl/p2p/config.go` sets `libp2p.Transport(tcp.NewTCPTransport)` and a `/tcp/` listen addr). The only QUIC path is the downloader's HTTP/3 transport, which is opt-in: `useHttp3 = os.Getenv("DOWNLOADER_HTTP3") != ""` (`db/downloader/downloader.go`). Default snapshot downloads use HTTP/2 over TCP, so `wrapConn` is never entered.
- Even when `DOWNLOADER_HTTP3=1` is set, current quic-go prints this at most **once per process** (`setBufferWarningOnce`) and it can be silenced with `QUIC_GO_DISABLE_RECEIVE_BUFFER_WARNING=true`. The 2024 spammy/repeating behavior is gone.

Note this was never an INFO-level log gated by `--log.console.verbosity`; it's a one-shot stderr line, so "not seeing it at INFO" is expected regardless.

**On buffer size / throughput:** the warning points at a real limit but there's no erigon code knob to fix it — `net.core.rmem_max`/`wmem_max` is a kernel ceiling and an unprivileged erigon cannot exceed it (quic-go already tries `SetReadBuffer` and, on Linux, `RCVBUFFORCE`, which needs `CAP_NET_ADMIN`). Operators enabling HTTP/3 should raise the sysctl (`net.core.rmem_max` / `net.core.wmem_max`, e.g. `7500000`) or pass it via the container/systemd for full QUIC throughput.
